src/Eppie.App/Eppie.App.ViewModels/DecentralizedAccountSettingsPageViewModel.cs

```diff
@@ -70,6 +122,22 @@ public static DecentralizedAccountSettingsModel Create(Account account)
         {
             return new DecentralizedAccountSettingsModel(account);
         }
+
+        public void UpdateAccount(Account account)
+        {
+            CurrentAccount = account ?? throw new ArgumentNullException(nameof(account));
+            Email = account.Email.Address;
+            SenderName = account.Email.Name;
+
+            if (account.Email.Network == NetworkType.Eppie)
+            {
+                SelectedNetwork = NetworkOptions[0];
+            }
+            else if (account.Email.Network == NetworkType.Bitcoin)
+            {
+                SelectedNetwork = NetworkOptions[1];
+            }
```

Comment:
This will be wrong when the order or number of items changes [Line: 103-104](https://github.com/Eppie-io/Eppie-App/pull/795/files#diff-b67a53638b214ceb6f8f6e484ab066cf4eb9775b0e2c54e80ce622190b359e10R103-R104)

Would you coordinate these pieces of code?

Reply:
Done.